#send the 

uint32[] command_list
---
bool result